src/main/java/com/example/user/repository/UserRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.example.user.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import com.example.user.domain.User;
+
+@Repository
+public interface UserRepository extends JpaRepository<User, int> {
```

Comment:
https://www.tuyano.com/index3?id=12626003
```suggestion
public interface UserRepository extends JpaRepository<User, Integer> {
```
型プリミティブ型ではなくオブジェクトにする必要がありそうなので